packages/form-js-viewer/test/spec/Form.spec.js

```diff
@@ -129,6 +131,45 @@ describe('Form', function() {
   });
 
 
+  (singleStartTheme ? it.only : it)('should render theme', async function() {
```

Comment:
I think instead of checking for the class maybe we could add a visual regression test. What do you think?

Reply:
I like that idea, good use case 👍

Reply:
I added an issue for me: https://github.com/camunda/team-hto/issues/225

I think we can handle this separately.